s3/aide.go

```diff
@@ -50,17 +61,13 @@ func (svc *Service) Put(key string, content io.Reader) (*s3.PutObjectOutput, err
 	return svc.svc.PutObject(in)
 }
 
+func (r *Object) ReadObject() (*io.ReadCloser, error) {
+	return r.Svc.Read(r.Key)
+}
```

Comment:
This is an exported type and needs documentation. You should call the method `Read` and return the concrete values satisfying `io.ReadCloser`. Or this is conflating the expected use of a read method. Also, `Object.ReadObject` stutters.

Reply:
Done